deployer/etherlime-ganache-deployer/etherlime-ganache-deployer.js

```diff
@@ -24,6 +24,18 @@ class EtherlimeGanacheDeployer extends JSONRPCDeployer {
 		this.nodeUrl = nodeUrl;
 	}
 
+	set privateKey(privateKey) {
```

Comment:
No need to make this setter. Instead we are looking for fully fledged method like `setPrivateKey`

Reply:
Oh! I get that now.